src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Data/FeatureLayerQuery/FeatureLayerQuery.xaml

```diff
@@ -0,0 +1,21 @@
+﻿<UserControl x:Class="ArcGISRuntime.Desktop.Samples.FeatureLayerQuery.FeatureLayerQuery"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013" 
+             mc:Ignorable="d" 
+             d:DesignHeight="300" d:DesignWidth="300">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <StackPanel  Orientation="Vertical" Grid.Row="0">
```

Comment:
Please follow default UI layout. Please refer to https://github.com/Esri/arcgis-runtime-samples-dotnet/blob/v.next/src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml